records_mover/db/postgres/loader.py

```diff
@@ -58,32 +67,30 @@ def load_from_fileobjs(self,
         table_obj = Table(table,
                           self.meta,
                           schema=schema,
-                          autoload=True,
-                          autoload_with=self.db)
+                          autoload_with=self.db_engine)
 
-        with self.db.engine.begin() as conn:
-            # https://www.postgresql.org/docs/8.3/sql-set.html
-            #
-            # The effects of SET LOCAL last only till the end of the
-            # current transaction, whether committed or not. A special
-            # case is SET followed by SET LOCAL within a single
-            # transaction: the SET LOCAL value will be seen until the end
-            # of the transaction, but afterwards (if the transaction is
-            # committed) the SET value will take effect.
-            date_style = f"ISO, {date_order_style}"
-            sql = f"SET LOCAL DateStyle = {quote_value(conn, date_style)}"
-            logger.info(sql)
-            conn.execute(sql)
+        # https://www.postgresql.org/docs/8.3/sql-set.html
+        #
+        # The effects of SET LOCAL last only till the end of the
+        # current transaction, whether committed or not. A special
+        # case is SET followed by SET LOCAL within a single
+        # transaction: the SET LOCAL value will be seen until the end
+        # of the transaction, but afterwards (if the transaction is
+        # committed) the SET value will take effect.
+        date_style = f"ISO, {date_order_style}"
+        sql = f"SET LOCAL DateStyle = {quote_value(None, date_style, db_engine=self.db_engine)}"
+        logger.info(sql)
+        self.db_conn.execute(text(sql))
```

Comment:
correct me if i am not understanding this right - so when we reference `self.db_conn` here, since it's None we'll create a new connection, and the opened_here flag will be set to True, because this class inherits from the mixin.

One question I have is is that gonna give the same transaction guarantees as the way the previous code was written? I know if you use a context manager like `with connection.begin()` it starts a transaction - but i don't think we are doing that here anymore so is that a problem??

Reply:
never mind then sounds good!